src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,108 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String DEFAULT_LOGIN = "userLogin";
+    private static final String DEFAULT_PASSWORD = "userPassword";
+    private static final int DEFAULT_AGE = 99;
+    private static final int WRONG_AGE = 17;
+    private static final String WRONG_PASSWORD = "small";
+
+    private RegistrationService registrationService;
+    private User user;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
```

Comment:
use can also initialize valid user in the setUp block, and then, in the tests just change some values that we need to check. e.g.
        validUser.setLogin(DEFAULT_LOGIN);
        validUser.setPassword(DEFAULT_PASSWORD);
        validUser.setAge(DEFAULT_AGE);

Reply:
ok